src/types/Currency.sol

```diff
@@ -37,42 +37,36 @@ library CurrencyLibrary {
 
     function transfer(Currency currency, address to, uint256 amount) internal {
         // implementation from
-        // https://github.com/transmissions11/solmate/blob/e8f96f25d48fe702117ce76c79228ca4f20206cb/src/utils/SafeTransferLib.sol
+        // https://github.com/Vectorized/solady/blob/0dde2a008d917aa8076f348eac2855edbe181cc0/src/utils/SafeTransferLib.sol
```

Comment:
this commit hasnt been audited from what i can see - lets update to the audited commit of 89101d53b7c8784cca935c1f2f6403639cee48b2

Reply:
and a more descriptive comment like `based on X` or `altered from X` as its not the same